department-of-veterans-affairs/va.gov-cms
# [Merge for July 13 deploy] VACMS-5669: Update URL patterns for Leadership Listing nodes.

## Description
See #5669

## Testing done
Visual, cypress, bulk update

## QA steps
- As an administrator, run bulk alias update on all content.
- Visit any location listing, and visually verify that the path is inherited from the parent item in the breadcrumb.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
